### ISSUE TITLE
Avoiding compression

### ISSUE THREAD

**Issue by pkfl** [TGSAI/mdio-python#176]
Is there a way to avoid compression (Blosc) when we run segy_to_mdio?  Will this work:

In src/mdio/segy/blocked_io.py

``` 
85     elif lossless is None:
86         trace_compressor = header_compressor = None
```

If yes, could you please support this in MDIO. That way people who do not want any compression can just ship lossless=None explicitly to segy_to_mdio. Thanks.

**Comment by tasansal** [comment]
Yes, we can support this. We need to refactor the CLI and parameters to be more user-friendly for compression, which this project can go into. If you have any ideas feel free to do a PR, we can discuss further there :)

**Comment by tasansal** [comment]
We currently don't have plans to implement this as part of MDIO v0. The v1 (next) release will allow full customization if users desire.
